tests/application/cms/test_views.py

```diff
@@ -570,6 +570,51 @@ def test_view_edit_measure_page(test_app_client, logged_in_rdu_user, stub_measur
     assert further_technical_information.text == "Further technical information"
 
 
+@pytest.mark.parametrize("measure_version, should_show_data_correction_radio", (("1.0", False), ("1.1", True)))
+def test_view_edit_measure_page_for_minor_update_shows_data_correction_radio(
+    test_app_client, logged_in_rdu_user, stub_measure_data, measure_version, should_show_data_correction_radio
+):
+    data_source = DataSourceFactory(
+        title="DWP Stats",
+        type_of_data=["SURVEY"],
+        source_url="http://dwp.gov.uk",
+        publication_date="15th May 2017",
+        note_on_corrections_or_updates="Note on corrections or updates",
+        purpose="Purpose of data source",
+    )
+
+    # We want to test a minor update to see that it shows the "update corrects data mistake" radio box
+    stub_measure_data["version"] = measure_version
+    measure_version = MeasureVersionFactory(status="DRAFT", data_sources=[data_source], **stub_measure_data)
+
+    response = test_app_client.get(
+        url_for(
+            "cms.edit_measure_version",
+            topic_slug=measure_version.measure.subtopic.topic.slug,
+            subtopic_slug=measure_version.measure.subtopic.slug,
+            measure_slug=measure_version.measure.slug,
+            version=measure_version.version,
+        )
+    )
+
+    assert response.status_code == 200
+    page = BeautifulSoup(response.data.decode("utf-8"), "html.parser")
+
+    assert page.h1.text.strip() == "Edit page"
+
+    update_corrects_data_mistake = page.find("div", id="update_corrects_data_mistake")
+    if should_show_data_correction_radio:
+        legend = update_corrects_data_mistake.find("legend")
+        labels = update_corrects_data_mistake.findAll("label")
+
+        assert legend.text.strip() == "Does this update correct a mistake in the data?"
+        assert labels[0].text.strip() == "Yes"
+        assert labels[1].text.strip() == "No"
```

Comment:
Should we assert that neither one is `checked` here too (ie that there is no default if you've not selected one yet)?

Reply:
Good idea. Done.